lib/graphql/testing/helpers.rb

```diff
@@ -41,7 +41,7 @@ def initialize(type_name:, field_name:)
 
       def run_graphql_field(schema, field_path, object, arguments: {}, context: {})
         type_name, *field_names = field_path.split(".")
-        dummy_query = GraphQL::Query.new(schema, context: context)
+        dummy_query = GraphQL::Query.new(schema, "query {}", context: context)
```

Comment:
I'm glad this passes _this_ test, but it might still fail somewhere because GraphQL says that selections _must_ exist (https://spec.graphql.org/draft/#sec-Validation.Fields). For example, I think GraphiQL attaches an error to this query:

![image](https://github.com/rmosolgo/graphql-ruby/assets/2231765/b8461d61-2eb3-46be-a4f7-96acdcddfd50)


Here's a minimal valid query: 

```suggestion
        dummy_query = GraphQL::Query.new(schema, "{ __typename }", context: context)
```

Reply:
Yep I wondered about whether I should add that. Added! Thanks for the review :)